presto-main/src/main/java/com/facebook/presto/execution/buffer/LazyOutputBuffer.java

```diff
@@ -190,6 +202,12 @@ public void setOutputBuffers(OutputBuffers newOutputBuffers)
         }
     }
 
+    private SpoolingOutputBuffer createSpoolingOutputBuffer(OutputBuffers outputBuffers)
+    {
+        TempStorage tempStorage = new LocalTempStorage(ImmutableList.of(createTempDir().toPath()), 1);
+        return new SpoolingOutputBuffer(taskId, taskInstanceId, outputBuffers, state, tempStorage, new DataSize(8, MEGABYTE), finalizerService);
```

Comment:
Minor question: Should there be a conf for the 8MB for the threshold?

Reply:
Yes, this has been added.